CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [2.1.1] - Unreleased
+- Add loaded function to modules, providing an async callback.
```

Comment:
@MichMich its ok but this its not follow the [Semantic Versioning](http://semver.org/). This a new feature.

What happend if we need release a new version with only bugfixes? The version name will 2.1.0.1? Will the first 2 always be maintained?